site/oauth2-examples-keycloak.md

```diff
@@ -109,10 +109,10 @@ rather than `legacy-token-key`.
 
 For backend applications which uses **Client Credentials flow** you create a **Client** with:
 
-* **Access Type** : `confidential`
+* **Access Type** : `public`
 * With all the other flows disabled: Standard Flow, Implicit Flow, Direct Access Grants
 * With **Service Accounts Enabled** on. If it is not enabled you do not have the tab `Credentials`
```

Comment:
Again enable/disable terminology used here which we want to avoid going forward, see the table here for more details: https://source.vmware.com/portal/pages/global-marketing/terminology-changes 
For the reference to "disabled" in the 3rd bullet point above, can we change it to "Turn off Standard Flow, Implicit Flow, and Direct Access Grants "


4th bullet point:  **Service Accounts Enabled** should change to **Service Accounts Activated** in PHASE 2 of inclusive language updates in code entities, more details about that in this issue: https://github.com/rabbitmq/rabbitmq-website/issues/1551

Are:  Standard Flow, Implicit Flow, Direct Access Grant UI entities?  if so, lets tag them appropriately, I think bold is being used to tag UI entities in OSS RabbitMQ docs right now, please use that here if so, thanks